commit 52cbe772a6e52a9b4d2989eef8acf489190ce94f (refs/changes/63/2563/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-09-13T15:16:54-07:00 (2 years, 1 month ago)
    
    DOCKER-1102 docker pull by manifest digest fails with an InternalError

diff --git a/package.json b/package.json
index 18faf99..fc67a5a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "workflow",
   "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "repository": {
@@ -17,7 +17,7 @@
     "uuid": "3.0.1",
     "async": "0.1.22",
     "vasync": "1.6.4",
-    "sdc-clients": "10.2.0",
+    "sdc-clients": "10.3.1",
     "verror": "1.6.0",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#e21ea53",
     "lstream": "0.0.4"
